rds-postgres/admin-login/main.tf

```diff
@@ -77,6 +70,23 @@ data "aws_db_instance" "this" {
   db_instance_identifier = var.identifier
 }
 
+data "aws_db_instance" "replica" {
+  count = var.replica_identifier ? 1 : 0
+
+  db_instance_identifier = var.replica_identifier
+}
+
 locals {
   full_name = join("-", ["rds-postgres", var.identifier])
+
+  base_value = {
+    dbname       = var.database_name
+    engine       = data.aws_db_instance.this.engine
+    host         = data.aws_db_instance.this.address
+    password     = var.initial_password
+    port         = tostring(data.aws_db_instance.this.port)
+    username     = var.username
+  }
+
+  initial_secret_value = var.replica_identifier ? local.base_value : merge(locals.base_value, { replica_host = data.aws_db_instance.replica.address })
```

Comment:
I might be reading this wrong, but is this condition backwards? It looks like it will merge the replica host only if `replica_identifier` is false/null.

Reply:
Nope you're not reading it wrong. it's backward. 🤦🏼‍♀️